cg/meta/archive/ddn/constants.py

```diff
@@ -49,3 +49,15 @@ class JobStatus(StrEnum):
     JobStatus.ON_VALIDATION,
     JobStatus.RUNNING,
 ]
+
+
+class MetadataFields(StrEnum):
+    CUSTOMER_NAME: str = "customer_name"
+    PREP_CATEGORY: str = "prep_category"
+    SAMPLE_NAME: str = "sample_name"
+    SEQUENCED_AT: str = "sequenced_at"
+    TICKET_NUMBER: str = "ticket_number"
+
+
+METADATA_FIELD_NAME = "metadataName"
+METADATA_FIELD_VALUE = "value"
```

Comment:
```suggestion
    TICKET_NUMBER: str = "ticket_number"
    NAME: str = "metadataName"
    VALUE: str = "value"
```

Feels like they belong here